woodwork/type_sys/inference_functions.py

```diff
@@ -104,3 +102,4 @@ def __call__(self, series: pd.Series) -> bool:
 
 email_address_func = InferWithRegex(lambda: ww.config.get_option('email_inference_regex'))
 url_func = InferWithRegex(lambda: ww.config.get_option('url_inference_regex'))
+ip_address_func = InferWithRegex(lambda: ("(" + ww.config.get_option('ipv4_inference_regex') + "|" + ww.config.get_option('ipv6_inference_regex') + ")"))
```

Comment:
Something like this might improve readability a bit:
```suggestion
ip_address_func = InferWithRegex(lambda: (
    "(" + ww.config.get_option('ipv4_inference_regex') +
    "|" + ww.config.get_option('ipv6_inference_regex') +")"
))
```